src/ContentHubClient.php

```diff
@@ -1029,9 +1029,12 @@ public function deleteClient($client_uuid = NULL) {
     $settings = $this->getSettings();
     $uuid = $client_uuid ?? $settings->getUuid();
     $response = $this->deleteEntity($uuid);
-    if (!$response || $response->getStatusCode() < 200 || $response->getStatusCode() >= 300) {
+    if (!$response) {
       throw new \Exception(sprintf("Entity with UUID = %s cannot be deleted.", $uuid));
     }
+    if ($response->getStatusCode() < 200 || $response->getStatusCode() >= 300) {
+      throw new \Exception(sprintf("Entity with UUID = %s cannot be deleted. Message: %s", $uuid, (string) $response->getBody()));
+    }
```

Comment:
We shouldn't throw an exception here in every case. For example if the client cdf is already missing, we should not halt the process just because there isn't one. This prevents the deletion of clients which don't have a corresponding client CDFs due to a purge operation. These are typically those, that are not connected to content hub anymore.

Reply:
Alright, I've removed exceptions thrown, unsuccessful responses will be handled in the calling methods.